workers/nintendo/buildbot.cfg

```diff
@@ -1,4 +1,13 @@
 builders = {
+    "ds": {
+        "configure_args": [
+            "--host=ds",
+            "--enable-mad",
```

Comment:
Prefer feature detection over explicitly passing feature flags like this, so that our releases are as consistent as possible across platforms. If the detection is broken such that an explicit flag is needed, that probably means something else is broken too.